### ISSUE TITLE
babelrc ignore doesn not work.

### ISSUE THREAD

**Issue by aruntk** [meteor/babel#16]
.babelrc

```json
{
  "ignore": [
    "imports/do-not-babel.js"
  ]
}
```

This does not seem to be working.

**Comment by krismeister** [comment]
Agreed, this is a very basic configuration file and it does not work. I'm  using `METEOR@1.4.4.3` 

This feature request would provide similar functionality to bablerc ignore https://github.com/meteor/meteor-feature-requests/issues/5

**Comment by zimme** [comment]
I do believe this should be solved separate from meteor/meteor-feature-requests#5. i.e. even if the build system get `imports/do-not-babel.js` the `ecmascript` package should be ignoring it instead.
